homepage/tests/user_likes_view_tests.py

```diff
@@ -0,0 +1,99 @@
+import pytest
+from homepage.models import User, Review, UserLikes
+from pytest_django.asserts import assertTemplateUsed
+
+
+@pytest.mark.django_db
+@pytest.fixture
+def db_data():
+    return User.objects.get(pk=2), Review.objects.get(pk=5)
+
+
+# check db when using view from reviews page
+@pytest.mark.django_db
+def test_view_toggle_like_from_reviews(client, db_data):
+    likes_num_before = db_data[1].likes_num
+    client.get(f'/like/{db_data[0].id}/{db_data[1].id}/', HTTP_REFERER='/reviews/')
+    assert UserLikes.objects.filter(user_id=db_data[0], review_id=db_data[1]).exists()
+    review = Review.objects.get(pk=5)  # pull updated
+    assert review.likes_num == likes_num_before + 1
+
+
+# check db when using view from single course page
+@pytest.mark.django_db
+def test_view_toggle_like_from_course(client, db_data):
+    likes_num_before = db_data[1].likes_num
+    client.get(f'/like/{db_data[0].id}/{db_data[1].id}/', HTTP_REFERER='/course/10111/')
+    assert UserLikes.objects.filter(user_id=db_data[0], review_id=db_data[1]).exists()
+    review = Review.objects.get(pk=5)  # pull updated
+    assert review.likes_num == likes_num_before + 1

```

Comment:
AFAICS these tests are the same except the HTTP_REFERER 
maybe you can do it without repetition with @pytest.mark.parametrize